packages/tests/src/nwaku.ts

```diff
@@ -16,14 +16,17 @@ import waitForLine from "./log_file.js";
 
 const log = debug("waku:nwaku");
 
-const WAKU_SERVICE_NODE_DIR =
-  process.env.WAKU_SERVICE_NODE_DIR ?? appRoot + "/nwaku";
-const WAKU_SERVICE_NODE_BIN =
-  process.env.WAKU_SERVICE_NODE_BIN ??
-  WAKU_SERVICE_NODE_DIR + "/build/wakunode2";
 const WAKU_SERVICE_NODE_PARAMS =
   process.env.WAKU_SERVICE_NODE_PARAMS ?? undefined;
 const NODE_READY_LOG_LINE = "Node setup complete";
+const NWAKU_VERSION = process.env.NWAKU_VERSION ?? "v0.16.0";
+const isGoWaku = process.env.IS_GO_WAKU !== undefined;
+
+const DOCKER_IMAGE =
+  process.env.LOCAL_IMAGE_NAME ||
+  (isGoWaku
+    ? "statusteam/go-waku:latest"
+    : `statusteam/nim-waku:${NWAKU_VERSION}`);
```

Comment:
I would suggest to just have **one** env variable `WAKUNODE_IMAGE` that defaults to `statusteam/nim-waku:v0.16.0`.
We can then set this image name in the CI for go-waku or nwaku master.

The various env variable just make it harder to understand what is going on and will lead to more changes if we decide to run a CI for go-waku latest **release** instead of latest **master**.